Gruntfile.js

```diff
@@ -69,7 +69,7 @@ module.exports = (grunt) => {
         options: {
           reporter: 'spec',
           require: './source/utils/winston.js',
-          timeout: 15000,
+          timeout: 35000,
           bail: true
         },
         src: 'nmclicktests/spec.*.js'
```

Comment:
Could you also remove the `nm` references ?

Reply:
Yes, I will probably rename this folder to just `clicktests` but I haven't checked if git is able to recognize this as a rename because of the big amount of changes within these files. Which is why I haven't done it yet to keep the changes as "clean" as possible for now.

Reply:
I changed it and indeed there are two many changes so the overall diff does not recognize the renames.